Signal/src/ViewControllers/MessagesViewController.m

```diff
@@ -1646,6 +1644,12 @@ - (UICollectionViewCell *)collectionView:(JSQMessagesCollectionView *)collection
     }
     cell.delegate = collectionView;
 
+    //    @property (nonatomic) uint64_t expiresAtSeconds;
+    //    @property (nonatomic) uint32_t expiresInSeconds;
+    //
+    //    @property (nonatomic, copy) NSDate *messageDate;
+    //    @property (nonatomic, retain) NSString *messageBody;
+
```

Comment:
Did you intend to leave this comment in? If so can you preface it with an explanation?